lab-kyle/test/testdata/data/123456

```diff
@@ -0,0 +1 @@
+{"id":123456,"content":"test test test"}
```

Comment:
Same goes for testdata as with data, it is better to gitignore and create relevant test data inside the tests themselves dynamically, removing anything they create at the end so your next test run gets a fresh start.